colossalai/shardformer/test/align_bert.py

```diff
@@ -0,0 +1,116 @@
+import os
```

Comment:
move this test to `tests/test_shardformer/test_bert.py`.

Reply:
done